public/JSONtoHTML.coffee

```diff
@@ -0,0 +1,211 @@
+# Variables
+xmlHttp = new XMLHttpRequest
+jsonFile = 'stdLib_base_test.json'
```

Comment:
use more descriptive names. The reader sees its a "json file". Use names having a semantic meaning - what this file is for? Maybe its a configuration file or somethig? Use variables to descirbe what this code is doing, not describe the code per se.